BHoM_Adapter/AdapterActions/Push.cs

```diff
@@ -94,10 +94,17 @@ public virtual List<object> Push(IEnumerable<object> objects, string tag = "", P
             List<Tuple<Type, PushType, IEnumerable<object>>> orderedObjects;
 
             if(m_AdapterSettings.HandleDependencies)
+            {
                 orderedObjects = Engine.Adapter.Query.GetDependencySortedObjects(objectsToPush, pushType, this);
+                if (m_AdapterSettings.HandlePriorities)
+                    orderedObjects = Engine.Adapter.Query.GetPrioritySortedObjects(orderedObjects, pushType, this);
+            }
             else
                 orderedObjects = objectsToPush.GroupBy(x => x.GetType()).Select(x => new Tuple<Type, PushType, IEnumerable<object>>(x.Key, pushType, x.Cast<object>())).ToList();
 
+
+
+
```

Comment:
Added here instead, if removed above

```suggestion

            if (m_AdapterSettings.HandlePriorities)
                orderedObjects = Engine.Adapter.Query.GetPrioritySortedObjects(orderedObjects, pushType, this);
```